lib/visitor-keys.js

```diff
@@ -234,38 +245,43 @@ const KEYS = {
     SequenceExpression: [
         "expressions"
     ],
+    SimpleCallExpression: [
+        "arguments",
+        "callee"
+    ],
```

Comment:
Nodes with `type: "SimpleCallExpression"` don't exist, but nodes with `type: "CallExpression"` do, and we need those here. When generating this object, we should probably use values of [type](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/64651581376f3b498f7d616863cfc626c8fbb11a/types/estree/index.d.ts#L341) properties instead of names of interfaces.

Reply:
With this approach:

1. This should cause `SimpleLiteral`, `BigIntLiteral`, and `RegExpLiteral` to be dropped from the new output, as apparently was the case earlier and I imagine still expected now (and merging isn't a problem for this script as they don't have any non-primitives to traverse).
2. `Directive` and `ExpressionStatement` present a similar situation, causing `Directive` to now be dropped as I think was expected, and though both have their own traversable property, it is the same name, `expression` (as makes sense).
3. `Comment` has two types `Line` and `Block` though I can make two types out of that (or ignore both).

I imagine there may be not too much work to adapt this to use `type`, and can then see if can get the expected results (minus the ordering of the keys which seems may not be as easy to do programmatically based solely on the AST).